pkg/virt-handler/isolation/isolation.go

```diff
@@ -134,15 +134,20 @@ func (s *socketBasedIsolationDetector) Detect(vm *v1.VirtualMachineInstance) (Is
 
 // standard golang libraries don't provide API to set runtime limits
 // for other processes, so we have to directly call to kernel
-func prLimit(pid int, limit uintptr, rlimit *unix.Rlimit) error {
+func setProcessMemoryLockLimit(pid int, currentLimit, maxLimit uint64) error {
+	rlimit := unix.Rlimit{
+		Cur: uint64(currentLimit),
+		Max: uint64(maxLimit),
+	}
 	_, _, errno := unix.RawSyscall6(unix.SYS_PRLIMIT64,
 		uintptr(pid),
-		limit,
-		uintptr(unsafe.Pointer(rlimit)), // #nosec used in unix RawSyscall6
+		uintptr(unix.RLIMIT_MEMLOCK), // #nosec used in unix RawSyscall6
```

Comment:
Why the comment of `#nosec` moved to a different command? Are you sure it is not needed now for both now?

Reply:
`#nosec` comments are added where unsafe pointers are used https://github.com/kubevirt/kubevirt/pull/4446
This comment no longer needed, I will remove it.

Reply:
I just realized we still need this comment for line 145, I will just move it